e2e-etl-python/files/Makefile

```diff
@@ -91,13 +71,7 @@ deploy: init-terraform plan
 .PHONY: deployment-test
 # Run (post-deployment) tests.
 deployment-test: install-test-deps
-	@$(call check_aws_credentials)
-
-	sh ./lib/scripts/createstackoutputs2yml.sh
-	inspec_profiles=$$(ls -1 ./test/integration); \
-	for profile in $$inspec_profiles; do \
-		JSON_VARS_FILE="$(PWD)/terraform.tfvars.json" bundle exec cinc-auditor exec ./test/integration/$$profile --no-create-lockfile --no-distinct-exit --input-file ./test/integration/$$profile/files/inputs-from-tfo-stack.yml --reporter=cli junit2:build/test-results/test/$$profile.xml json:reports/install/data/inspec/post-install/$$profile.json --target aws://; \
-	done
+	#Removed for the e2e-etl-python QS
```

Comment:
this comment won't make sense to the end user and doesn't provide any context or important information to them